Lil-Peep/src/styles/modules/pdp.scss

```diff
@@ -0,0 +1,113 @@
+.product-container {
+  background-color: #f7f8f9;
```

Comment:
Color literals like `#f7f8f9` should only be used in variable declarations; they should be referred to via variable everywhere else.